edgePy/DGEList.py

```diff
@@ -58,9 +61,14 @@ def __init__(
         groups_in_dict: Optional[Dict] = None,
         to_remove_zeroes: Optional[bool] = False,
         filename: Optional[str] = None,
+        current_type: Optional[str] = None,
+        current_log: Optional[bool] = False,
     ) -> None:
 
         self.to_remove_zeroes = to_remove_zeroes
+        self.current_data_format = current_type
+        self.log = current_log
```

Comment:
I see no reason to have different names for parameter and property. Also: log is not descriptive. Maybe use
is_log_transformed or so

Reply:
done!